docs/version0.40.md

```diff
@@ -54,6 +55,10 @@ For more information, see the [Cloud optimizations](introduction.md#cloud-optimi
 
 Persistent shared caches are now supported on z/OS systems. The default cache type on z/OS is still non-persistent. You can use the [`-Xshareclasses:persistent`](xshareclasses.md#persistent) option to switch to persistent shared cache.
 
+### Support for EC key generation algorithm and XDH key agreement and generation algorithm by using native OpenSSL library added
+
+The EC key generation algorithms can now use native OpenSSL library for all OpenJDK versions. The XDH key agreement and generation algorithms can also use native OpenSSL library. For more information, see [`-Djdk.nativeECKeyGen`](djdknativeeckeygen.md), [`-Djdk.nativeXDHKeyAgreement`](djdknativexdhkeyagreement.md), and [`-Djdk.nativeXDHKeyGen`](djdknativexdhkeygen.md) .
```

Comment:
I think it's singular. As in algorithm, not algorithms. That's for the EC key generation. The plural in XDH makes sense since it refers to agreement and generation.